app/gimletlet/app-sidecar-emulator.toml

```
@@ -0,0 +1,258 @@
+name = "gimletlet-sidecar-emulator"
+target = "thumbv7em-none-eabihf"
+board = "gimletlet-2"
+chip = "../../chips/stm32h7.toml"
```

Now that #539 is merged, you'll need to edit this line to read
```suggestion
chip = "../../chips/stm32h7"
```
(i.e. no trailing `.toml`)